internal/pkg/service/stream/api/mapper/sink.go

```diff
@@ -17,25 +19,44 @@ func (m *Mapper) NewSinkStatisticsTotalResponse(result statistics.Aggregated) (r
 	}
 }
 
+func (m *Mapper) NewSinkFile(file model.File) *stream.SinkFile {
+	return &stream.SinkFile{
+		State:       file.State,
+		OpenedAt:    file.OpenedAt().String(),
+		ClosingAt:   timeToString(file.ClosingAt),
+		ImportingAt: timeToString(file.ImportingAt),
+		ImportedAt:  timeToString(file.ImportedAt),
+	}
+}
+
+func (m *Mapper) NewSinkFileStatistics(result *statistics.Aggregated) *stream.SinkFileStatistics {
+	return &stream.SinkFileStatistics{
+		Total: mapValueToLevel(result.Total),
+		Levels: &stream.Levels{
+			Local:   mapValueToLevel(result.Local),
+			Staging: mapValueToLevel(result.Staging),
+			Target:  mapValueToLevel(result.Target),
+		},
+	}
+}
+
 func mapValueToLevel(value statistics.Value) *stream.Level {
 	if value.RecordsCount == 0 {
 		return nil
 	}
 
 	return &stream.Level{
-		FirstRecordAt:    timeValueToPointer(value.FirstRecordAt),
-		LastRecordAt:     timeValueToPointer(value.LastRecordAt),
+		FirstRecordAt:    timeToString(&value.FirstRecordAt),
+		LastRecordAt:     timeToString(&value.LastRecordAt),
 		RecordsCount:     value.RecordsCount,
 		UncompressedSize: uint64(value.UncompressedSize),
 	}
 }
 
-func timeValueToPointer(time utctime.UTCTime) *string {
-	var result *string
-	if !time.IsZero() {
-		value := time.String()
-		result = &value
+func timeToString(time *utctime.UTCTime) *string {
```

Comment:
`timeToStringPointer` Someone can use it and expects that it returns `""` as default.